core/server/worker/src/main/java/alluxio/worker/block/BlockMasterSync.java

```diff
@@ -109,7 +113,7 @@ private void registerWithMaster() throws IOException {
     mMasterClient.register(mWorkerId.get(),
         storageTierAssoc.getOrderedStorageAliases(), storeMeta.getCapacityBytesOnTiers(),
         storeMeta.getUsedBytesOnTiers(), storeMeta.getBlockListByStorageLocation(),
-        storeMeta.getLostStorage(), configList);
+        storeMeta.getLostStorage(), configList, mStartTime, ProjectConstants.VERSION);
```

Comment:
For the version, maybe add the revision too? `version` is like `2.6.0` and revision is the hashcode which means the commit in the git repo. You can see how that is got from the `alluxio.cli.Version` command.

Reply:
Good suggestion, it helps.